karapace/master_coordinator.py

```diff
@@ -212,6 +222,16 @@ def init_schema_coordinator(self) -> None:
         )
         self.schema_coordinator_ready.set()
 
+    def get_coordinator_status(self) -> SchemaCoordinatorStatus:
+        generation = self.sc.generation() if self.sc is not None else None
+        return SchemaCoordinatorStatus(
+            is_primary=self.sc.are_we_master if self.sc is not None else None,
+            is_primary_eligible=cast(bool, self.config["master_eligibility"]),
```

Comment:
This is available from `SchemaCoordinator.master_eligibility`.
https://github.com/aiven/karapace/blob/2ab4e832549274442ac13a175c821a5e9f27890b/karapace/master_coordinator.py#L73

Reply:
However `SchemaCoordinator` might not be initialized in the beginning ie. `self.sc is None`...